<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QPushButton" name="hand_pushButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>手动生成</string>
   </property>
  </widget>
  <widget class="QPushButton" name="auto_pushButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>230</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>自动生成</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="x_textEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="y_textEdit">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>120</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="close_pushButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>350</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>关闭地图</string>
   </property>
  </widget>
  <widget class="QPushButton" name="DFS_pushButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>270</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>DFS寻路</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>20</y>
     <width>171</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>迷宫的设计与实现</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>100</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>任务要求：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>370</y>
     <width>121</width>
     <height>19</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>玩法说明：</string>
   </property>
  </widget>
  <widget class="QLabel" name="star_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>420</y>
     <width>21</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>420</y>
     <width>69</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>起点</string>
   </property>
  </widget>
  <widget class="QLabel" name="path_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>480</y>
     <width>21</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="wall_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>510</y>
     <width>21</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_22">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>450</y>
     <width>69</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>终点</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>480</y>
     <width>69</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>路</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>510</y>
     <width>69</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>墙</string>
   </property>
  </widget>
  <widget class="QLabel" name="through_path_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>540</y>
     <width>21</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>540</y>
     <width>69</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>经过路径</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>420</y>
     <width>171</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>鼠标左键设置路或者墙：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>450</y>
     <width>231</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>如果是路则变墙，如果是墙则变路</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>480</y>
     <width>181</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>鼠标有键设置起点和终点：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_16">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>510</y>
     <width>261</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>没有起点设置起点，没有终点设置终点</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_17">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>540</y>
     <width>321</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>如果有起点点击起点则取消起点，否则设置终点</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_18">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>560</y>
     <width>321</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>如果有终点点击终点则取消终点，否则设置起点</string>
   </property>
  </widget>
  <widget class="QLabel" name="end_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>449</y>
     <width>21</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>153</y>
     <width>276</width>
     <height>99</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>(1)具有图形化界面</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>(2)给定迷宫或者随机生成迷宫</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>(3)演示迷宫下的从入口到出口的探索过程</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>(4)生成路径</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>310</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>A星算法</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
